### ISSUE TITLE
MeasureControl plugin scrolls map unwantedly

### ISSUE THREAD

**Issue by fcenzato** [python-visualization/folium#1737]
**Describe the bug**
When trying to measure a distance with MeasureControl plugin, every time I select a point on the map, the map automatically scrolls, resulting in the point being not in the location I want it to be. This way, the plugin can't be used. See attached video for an example.

**To Reproduce**
Create a map with the following code, and open the saved file in a browser (Chrome or Edge):
```
import folium
from folium.plugins import MeasureControl

base_map = folium.Map()
MeasureControl().add_to(base_map)
base_map.save(outfile="test_map.html")
```

**Expected behavior**
Whenever you click to add a mark on the map, that mark should be at the point you clicked, not at a random location from an unwanted scrolling of the map

**Environment:**
 - Browser: Edge and Chrome (same behaviour)
 - Jupyter Notebook or html files: same behaviour in both
 - Python version: 3.10.9
 - folium version: 0.14.0
 - branca version: 0.6.0

**Possible solutions**
As a workaround, I have downgraded folium to version 0.12.1, in which the plugin works correctly

https://user-images.githubusercontent.com/99724166/224942584-30b43429-6178-4004-921c-fe3c1da82f38.mp4

**Comment by iacisme** [comment]
I'm having the same issue in the latest folium.

This issue also exists in leafmap: https://github.com/jupyter-widgets/ipyleaflet/issues/1076 so might it be something upstream?

I've also noticed marker clusters are acting funny as well, same as in ipyleaflet.

**Comment by Conengmo** [comment]
Unfortunately this is an issue in the upstream JS plugin we use: https://github.com/ljagis/leaflet-measure/issues/171. This package is currently abandoned, so there's no fix.

A temporary work around is to use an older Leaflet version:

```
m = folium.Map()
m.default_js.append(('leaflet', 'https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.js'))
m.default_css.append(('leaflet_css', 'https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.css'))
```

I'd like to think about how we should solve this going forward:

- Wait and hope for the original repo to get a new maintainer and the plugin to get updated.
- Adopt a different plugin inside the `MeasureControl` class.
- Remove the `MeasureControl` class.

If any of you involved in this issue know of an alternative JS plugin that we could adopt, I'd love to hear it!

**Comment by Conengmo** [comment]
@iacisme, @fcenzato, could either of you confirm the PR I opened does fix the problem for you as well? If so, I'll merge it and it'll be included in the next Folium release.

**Comment by fcenzato** [comment]
@Conengmo I tried installing that PR and the issue is not there anymore.

**Comment by Conengmo** [comment]
Great! I'll merge that PR, so the fix will be available to anyone who install Folium from the main git branch. It will also be included in the next release, v0.15.0.

**Comment by iacisme** [comment]
Sorry for being late to the party, thank you for resolving this issue!
